src/App.js

```diff
@@ -4,7 +4,7 @@ import Header from './components/header/Header';
 import Rockets from './routes/rockets/Rockets';
 import Profile from './routes/profile/Profile';
 // eslint-disable-next-line import/extensions
```

Comment:
- [ ] Kindly remove the disabled linter, and resolve it manually. You can get help from [here](https://stackoverflow.com/questions/59265981/typescript-eslint-missing-file-extension-ts-import-extensions) or just type the error in google for more solutions.